src/stories/Breadcrumb/BreadcrumbMeta.ts

```diff
@@ -8,16 +14,20 @@ export const meta = {
     },
   },
   argTypes: {
-    items: {
-      name: "items",
-      description: "List of page's title",
+    elements: {
```

Comment:
Let's disable elements from the control panel as it is not part of the component

` elements: {
      table: {
        disable: true,
      },
    },
    `